docs/configuration-parameters.md

```diff
@@ -142,7 +142,7 @@ This section provides the details of all the configurations required for the Car
 | carbon.scheduler.min.registered.resources.ratio | 0.8 | Specifies the minimum resource (executor) ratio needed for starting the block distribution. The default value is 0.8, which indicates 80% of the requested resource is allocated for starting block distribution. The minimum value is 0.1 min and the maximum value is 1.0. |
 | carbon.detail.batch.size | 100 | The buffer size to store records, returned from the block scan. In limit scenario this parameter is very important. For example your query limit is 1000. But if we set this value to 3000 that means we get 3000 records from scan but spark will only take 1000 rows. So the 2000 remaining are useless. In one Finance test case after we set it to 100, in the limit 1000 scenario the performance increase about 2 times in comparison to if we set this value to 12000.<br /><br /> **NOTE** The minimum batch size allowed is 100 and maximum batch size allowed by this property is 1000. |
 | carbon.enable.vector.reader | true | Spark added vector processing to optimize cpu cache miss and there by increase the query performance. This configuration enables to fetch data as columnar batch of size 4*1024 rows instead of fetching data row by row and provide it to spark so that there is improvement in  select queries performance. |
-| carbon.enable.mv | true | Whether to rewrite the query plan based on the materialized views, Default value is true |
```

Comment:
if any open source community user has been using MV this will impact thier upgrade, hope the upgrade document is clear and upgrade steps are ensured

Reply:
its good to prepare steps. this i will handle in separate PR.